advocacy_docs/edb-postgres-ai/console/getstarted/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: EDB Postgres AI Console - Get started
+navTitle: Get started
+description: Get started with the EDB Postgres AI Console.
+deepToC: true
+---
+
+The EDB Postgres® AI Console is a web-based user interface that provides a single pane of glass for managing and monitoring EDB Postgres AI Database Cloud Service and EDB Postgres AI databases. The EDB Postgres AI Console provides a unified view of the EDB Postgres AI Database Cloud Service and EDB Postgres AI databases, allowing users to manage and monitor their databases, users, and resources from a single interface.
```

Comment:
> EDB Postgres AI Database Cloud Service and EDB Postgres AI databases

And non-EDB Postgres like RDS or any random PG you built from source

Reply:
Done